src/mono/sample/iOS-NativeAOT/Program.csproj

```diff
@@ -49,7 +48,7 @@
       <PropertyGroup>
         <IlcPath>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(HostOS).$(BuildArchitecture).$(CoreCLRConfiguration)', 'ilc'))</IlcPath>
```

Comment:
I wonder - if we change this to point to the ilc_crossarch project, would we be able to stop building the ILC project when targeting iOSlike?

Reply:
Right, this will be updated once the cross-build is resolved.